web-services/deploy/application/src/main/wildfly/add-datawave-configuration.cli

```diff
@@ -121,6 +121,15 @@ module add --name=com.mysql.driver --dependencies=javax.api,javax.transaction.ap
 /system-property=dw.remoteAccumuloService.host:add(value=${accumulo.remoteservice.host})
 /system-property=dw.remoteAccumuloService.port:add(value=${accumulo.remoteservice.port})
 
+# QueryMetric microservice config
+/system-property=dw.remoteQueryMetricService.enabled:add(value=${querymetric.remoteservice.enabled})
```

Comment:
I can't find this property being used.

Reply:
Agree -- removed those lines and the property from default.  Also added lines to remove-datawave-configuration.cli